metapool/tests/test_plate.py

```diff
@@ -261,6 +269,15 @@ def test_validate_plate_metadata_full(self):
         pd.testing.assert_frame_equal(validate_plate_metadata(self.metadata),
                                       expected)
 
+    def test_validate_plate_metadata_full_extra_column(self):
```

Comment:
Does this means that it actually adds, just fine, any extra columns?

Reply:
Correct. Additional unexpected columns can be added and a warning will be given but the function will otherwise operate as intended. The unexpected columns will appear in the plate-metadata result.